packages/millicast-viewer-demo/src/viewer.js

```diff
@@ -72,6 +72,11 @@ const newViewer = () => {
 
   millicastView.on('onMetadata', (metadata) => {
     console.log('Metadata event:', metadata)
+    if (metadata.uuid === DOLBY_SEI_DATA_UUID) {
+      console.log('received SEI unregistered messsage', metadata.unregistered)
+    } else if (metadata.uuid === DOLBY_SEI_TIMESTAMP_UUID) {
+      console.log('received SEI unregistered messsage', metadata.unregistered)
+    }
```

Comment:
Ah thanks!

Reply:
The reason I put this here is so that any external user who pulls down the SDK has a working sample on how to integrate metadata and what they'd need to do to build their own viewer/experience on top of the SDK

Reply:
I understand that, just to keep in mind, that probably if the UUID received is the timestamp one, the `unregistered` attribute will be undefined